docs/estela-cli/quickstart.md

```diff
@@ -38,7 +38,20 @@ Successful login. API Token stored in ~/.estela.yaml.
 ```
 
 This will save your estela API key to the file `~/.estela.yaml`, and it is
-needed to access projects associated with your account.
+needed to access projects associated with your account.  
+
+**_Note:_** If you have installed Estela locally run the following command to get the _EXTERNAL-IP_ for the django-api:
+```bash
+$ kubectl get all
```

Comment:
Please change the command to: `$ minikube ssh 'grep host.minikube.internal /etc/hosts | cut -f1'`. By using this command we get the ip directly.